template-for-repository/models/README.md

```diff
@@ -1,6 +1,6 @@
 CBMC proof source code
```

Comment:
This renaming of sources to models is a bit gratuitous since the MQTT and HTTP folks have already started populating sources and includes with project code.  If it is likely people will think cbmc/sources is part of the source tree, it is equally likely that people will think cbmc/include is part of the source tree.  It might be worth using the names PROOF_SOURCE=cbmc/sources and PROOF_INCLUDE=cbmc/include in the Makefile.common.

Reply:
Done